File: frontend/lib/neighborhoods.ts
export interface NeighborhoodMeta {
  value: string;
  label: string;
  lat: number;
  lng: number;
}

export const NEIGHBORHOODS: NeighborhoodMeta[] = [
  { value: "nachlaot",      label: "Nachlaot",       lat: 31.7812, lng: 35.2091 },
  { value: "rehavia",       label: "Rehavia",        lat: 31.7726, lng: 35.2124 },
  { value: "katamon",       label: "Katamon",        lat: 31.7593, lng: 35.2121 },
  { value: "talbia",        label: "Talbia",         lat: 31.7648, lng: 35.2139 },
  { value: "city_center",   label: "City Center",    lat: 31.7781, lng: 35.2244 },
  { value: "german_colony", label: "German Colony",  lat: 31.7567, lng: 35.2201 },
  { value: "baka",          label: "Baka",           lat: 31.7527, lng: 35.2220 },
  { value: "ramot",         label: "Ramot",          lat: 31.8220, lng: 35.1940 },
  { value: "pisgat_zeev",   label: "Pisgat Ze'ev",  lat: 31.8278, lng: 35.2440 },
  { value: "neve_yaacov",   label: "Neve Ya'acov",  lat: 31.8428, lng: 35.2388 },
  { value: "gilo",          label: "Gilo",           lat: 31.7297, lng: 35.1822 },
  { value: "har_homa",      label: "Har Homa",       lat: 31.7197, lng: 35.2047 },
  { value: "talpiot",       label: "Talpiot",        lat: 31.7503, lng: 35.2285 },
  { value: "arnona",        label: "Arnona",         lat: 31.7453, lng: 35.2204 },
  { value: "kiryat_yovel",  label: "Kiryat Yovel",  lat: 31.7680, lng: 35.1852 },
  { value: "ein_karem",     label: "Ein Karem",      lat: 31.7663, lng: 35.1608 },
  { value: "har_nof",       label: "Har Nof",        lat: 31.7835, lng: 35.1742 },
  { value: "beit_hakerem",  label: "Beit HaKerem",  lat: 31.7809, lng: 35.1890 },
  { value: "kiryat_moshe",  label: "Kiryat Moshe",  lat: 31.7870, lng: 35.1985 },
];

export const NEIGHBORHOOD_MAP = new Map(NEIGHBORHOODS.map((n) => [n.value, n]));

File: backend/neighborhoods.py
from typing import Dict, Tuple

# Approximate center coordinates for each Jerusalem neighborhood
# (latitude, longitude)
NEIGHBORHOOD_CENTERS: Dict[str, Tuple[float, float]] = {
    "nachlaot":      (31.7812, 35.2091),
    "rehavia":       (31.7726, 35.2124),
    "katamon":       (31.7593, 35.2121),
    "talbia":        (31.7648, 35.2139),
    "city_center":   (31.7781, 35.2244),
    "german_colony": (31.7567, 35.2201),
    "baka":          (31.7527, 35.2220),
    "ramot":         (31.8220, 35.1940),
    "pisgat_zeev":   (31.8278, 35.2440),
    "neve_yaacov":   (31.8428, 35.2388),
    "gilo":          (31.7297, 35.1822),
    "har_homa":      (31.7197, 35.2047),
    "talpiot":       (31.7503, 35.2285),
    "arnona":        (31.7453, 35.2204),
    "kiryat_yovel":  (31.7680, 35.1852),
    "ein_karem":     (31.7663, 35.1608),
    "har_nof":       (31.7835, 35.1742),
    "beit_hakerem":  (31.7809, 35.1890),
    "kiryat_moshe":  (31.7870, 35.1985),
}

NEIGHBORHOOD_DISPLAY_NAMES: Dict[str, str] = {
    "nachlaot":      "Nachlaot",
    "rehavia":       "Rehavia",
    "katamon":       "Katamon",
    "talbia":        "Talbia",
    "city_center":   "City Center",
    "german_colony": "German Colony",
    "baka":          "Baka",
    "ramot":         "Ramot",
    "pisgat_zeev":   "Pisgat Ze'ev",
    "neve_yaacov":   "Neve Ya'acov",
    "gilo":          "Gilo",
    "har_homa":      "Har Homa",
    "talpiot":       "Talpiot",
    "arnona":        "Arnona",
    "kiryat_yovel":  "Kiryat Yovel",
    "ein_karem":     "Ein Karem",
    "har_nof":       "Har Nof",
    "beit_hakerem":  "Beit HaKerem",
    "kiryat_moshe":  "Kiryat Moshe",
}
